src/main/kotlin/it/pagopa/wallet/audit/LoggingEvent.kt

```diff
@@ -18,17 +18,15 @@ data class WalletDeletedEvent(val walletId: String) : LoggingEvent()
 
 data class SessionWalletCreatedEvent(val walletId: String) : LoggingEvent()
 
-data class WalletPatchEvent(val walletId: String) : LoggingEvent()
+data class WalletPatchEvent(
```

Comment:
Cause we are refactoring wallet events, what about giving name near "business" meaning? 

For example this represent an update to wallet applications e.g. WalletApplicationsUpdated